bin/handler-sensu.rb

```diff
@@ -81,19 +81,21 @@ def filter_repeated; end
 
   def handle
     client = @event['client']['name']
+    check = @event['check']['name']
     remediations = @event['check']['remediation']
     occurrences = @event['occurrences']
+    trigger_on = @event['check']['trigger_on']
     severity = @event['check']['status'].to_i
-    puts "REMEDIATION: Evaluating remediation: #{client} "\
+    puts "REMEDIATION: Evaluating remediation: #{client} #{check} "\
          "#{remediations.inspect} #=#{occurrences} sev=#{severity}"
 
     remediation_checks = parse_remediations(remediations, occurrences, severity)
 
     # at some point we should come back and remove the old default subscription of [client]
-    subscribers = @event['check']['trigger_on'] ? @event['check']['trigger_on'] : ['client:' + client, client]
+    subscribers = trigger_on ? @event['check']['trigger_on'] : "client:#{client}"
```

Comment:
This makes it a breaking change, the thought here was to provide a fallback as described here: https://github.com/sensu-plugins/sensu-plugins-sensu/pull/42#issuecomment-332114024 does this not work as expected?

Reply:
ok. I'll keep this: ['client:' + client, client]